### ISSUE TITLE
The losses used in the two training stages seems to be wrong?

### ISSUE THREAD

**Issue by Charles-Xie** [enlighten0707/Body-Part-Map-for-Interactiveness#5]
Thanks for sharing your awesome work!
Something about the training losses bothers me recently.

Your network consists of a feature extractor, a box detector, an interactiveness classifier and a verb classifier. Training has 2 stages.
According to Section 3.5 in your paper, in the 1st training stage, the feature extractor, box detector & interactiveness classifier is trained. In the 2nd stage, the feature extractor, box detector & verb classifier is trained.
However, in the 2nd stage, **the interactiveness classifier is not trained with any loss, and the feature extractor is obviously not frozen** since it is also trained as you stated. This will lead to the corruption of the interactiveness classifier parameters. 

How do you solve this problem? Is there something not mentioned in the paper or anything I'm not understanding correctly?

I look forward to your reply. 🙏 Thanks!

<img width="654" alt="screenshot from the paper" src="https://user-images.githubusercontent.com/24546725/197530178-aa9b399a-707f-4435-8596-97401602efd8.png">

**Comment by enlighten0707** [comment]
In the 1-st stage, we obtain detection results with interactiveness classification. Then in the 2-nd stage, we obtain detection results with verb classification without interactiveness. 
Thus, we finally get two predicted sets with different detection results. We match the two sets and then evaluate. It is detailed in Suppl.C in our [paper](https://arxiv.org/pdf/2207.14192v2.pdf).
